plugin/secrets_service_account_key.go

```diff
@@ -213,13 +297,90 @@ func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleS
 	}
 
 	resp := b.Secret(SecretTypeKey).Response(secretD, internalD)
-	resp.Secret.Renewable = true
+	resp.Secret.Renewable = false
```

Comment:
Why are we setting the renewable flag to false?

Reply:
Because, if we keep the key renewable, there's a high chance that we'll hit the number of service account key limit again.